website/docs/patterns/form-patterns/partials/validation/validation.md

```diff
@@ -72,32 +99,16 @@ Validating a form on submission can occur on the client or the server and can be
 
 ### Delayed validation
 
-Delayed validation occurs on the client and refers to validating the field after a lapse in keystrokes or a specific interval of time, e.g., `500ms` or 0.5 seconds. Once the user has stopped input into the field or after the interval of time has expired, the field is validated without an `onblur` event occurring.
+Delayed validation occurs on the client side and refers to validating the field after a lapse in keystrokes or a specific interval of time, e.g., `500ms` or 0.5 seconds. Once the user has stopped input into the field or after the interval of time has expired, the field is validated without an `onblur` event occurring.
```

Comment:
```suggestion
Delayed validation occurs on the client-side and refers to validating the field after a lapse in keystrokes or a specific interval of time, e.g., `500ms` or 0.5 seconds. Once the user has stopped typing into the field or after the interval of time has expired, the field is validated without an `onblur` event occurring.
```

Reply:
When referring to events that happen on the client, "client side" and "client-side" have somewhat different grammatical uses.

```suggestion
Delayed validation occurs client-side and refers to validating the field after a lapse in keystrokes or a specific interval of time, e.g., `500ms` or 0.5 seconds. Once the user has typing into the field or after the interval of time has expired, the field is validated without an `onblur` event occurring.
```